improver/utilities/load.py

```diff
@@ -38,6 +39,16 @@
     enforce_coordinate_ordering, merge_cubes)
 
 
+@contextlib.contextmanager
+def monkeypatched(object, name, patch):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'object'](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5023279)